core/domain/state_domain.py

```diff
@@ -149,6 +149,28 @@ def validate(self, interaction, exp_param_specs_dict):
 
         self.outcome.validate()
 
+    @classmethod
+    def convert_html_in_answer_group(cls, answer_group_dict, conversion_fn):
```

Comment:
maybe add `_dict` at the end of the function name to clarify that it converts the dict? Not sure, any thoughts?

Reply:
I don't think it's required, as it's mentioned in the docstring.
But still if you insist I'll add it.